cmd/disaster-recovery/disaster_recovery_test.py

```
@@ -0,0 +1,335 @@
+                    print(f"[SUCCESS] {json.dumps(data, indent=2)}")
+                except json.JSONDecodeError:
+                    print(f"[SUCCESS] Response body: {response.text}")
+            else:
+                print(f"[ERROR] {response.status_code}: {response.text}")
+
+        except Exception as e:
+            print(f"[ERROR] Request failed: {e}")
+
+    def run_continuous(self, interval=15):
+        """Run the toggle operation continuously"""
+        print(f"=== Quickstarts Disaster Recovery Test ===")
```

Ruff (0.15.15)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)